pages/docs/reference/building-mpp-with-gradle.md

```diff
@@ -392,7 +394,11 @@ There are target presets that one can apply using the preset functions, as shown
 following target platforms:
 
 * `jvm` for Kotlin/JVM;
-* `js` for Kotlin/JS;
+* `js` for Kotlin/JS; requiring a choice for the execution environment in their configuration block (see also [Setting up a Kotlin/JS project](/docs/tutorials/javascript/setting-up.html#setting-up-using-the-kotlin-multiplatform-plugin)):
```

Comment:
I've incorporated this change into new MPP docs. 
You can merge these changes. But I will overwrite them next week.

Reply:
Works for me :)